src/NServiceBus.Learning.AcceptanceTests/NServiceBus.Learning.AcceptanceTests.csproj

```diff
@@ -22,7 +22,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Include="../NServiceBus.AcceptanceTests/**/*.cs" Exclude="../NServiceBus.AcceptanceTests/obj/**/*.*" />
+    <Compile Include="../NServiceBus.AcceptanceTests/**/*.cs" Exclude="../NServiceBus.AcceptanceTests/obj/**/*.*;../NServiceBus.AcceptanceTests/Core/**/*.cs" />
```

Comment:
Are you sure this is correct? I thought the intention was to let the Core tests also run on the learning persistence. At least that is how they have been setup since ages as far as I recall

Reply:
There is no outbox available for the learningpersistence so the other option would be to disable just the tests we add here but that felt like a bit of a hack?

Reply:
Also not that the core tests run for the  acceptance testing persistence which is a copy paste of the learning + a custom outbox

Reply:
and another option is to make the learning persister have an outbox, perhaps this is better? (but larger scope)